src/org/opensolaris/opengrok/analysis/pascal/PascalSymbolTokenizer.lex

```diff
@@ -37,44 +38,57 @@ import org.opensolaris.opengrok.analysis.JFlexSymbolMatcher;
     yyline = 1;
 %init}
 %unicode
+%ignorecase
 %int
-%include CommonLexer.lexh
 %char
+%include CommonLexer.lexh
 
-Identifier = [a-zA-Z_] [a-zA-Z0-9_]*
-
-%state STRING COMMENT SCOMMENT QSTRING
+%state COMMENT PCOMMENT SCOMMENT QSTRING
 
+%include Common.lexh
+%include Pascal.lexh
 %%
 
 <YYINITIAL> {
-{Identifier} {String id = yytext();
-                if(!Consts.kwd.contains(id)){
+\&{Identifier}    {
+    String id = yytext();
+    onSymbolMatched(id.substring(1), yychar + 1);
+    return yystate();
+}
+{Identifier}    {
+    String id = yytext();
+    if (!Consts.kwd.contains(id.toLowerCase(Locale.ROOT))) {
```

Comment:
so I am wondering if we shouldn't set UTF-8 everywhere ...

Reply:
Yes, I have an upcoming PR (a965f83a) where I reviewed Locale usage.

The most compelling case is for Turkish users, where the [Turkish locale bug](http://mattryall.net/blog/2009/02/the-infamous-turkish-locale-bug) can surface and make very non-intuitive case changes and result in e.g. "Turkish characters are not displayed properly on diff and search results" (issue #1737) and "Turkish character search problem regarding to specific character I" (issue #1768).